### PR TITLE
feat: add typing in some functions with corresponding test

### PR DESCRIPTION
**Pull request by roccia**
add typing in some files, also created a test for jina.executors.encoders.helper class (only for reduce_mean function,since the return value in other functions in helper are the same)

### CODE REVIEW

File: jina/helper.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def print_load_table(load_stat):
         default_logger.info('\n'.join(load_table))
 
 
-def print_load_csv_table(load_stat):
+def print_load_csv_table(load_stat: Dict[str, List[any]]):
```
Comment (JoanFM) [inline jina/helper.py:76]: Shouldn't it be Any with capital A?

**Comment by JoanFM** [comment]
Thank you very much for your contribution. Would it be possible for you to add also type hints for the return values?

Maybe you can use the help of mypy and MonkeyType?

File: jina/helper.py (hunk 1 above)
Comment (roccia) [reply jina/helper.py:76]: yes, my bad,wil fix it asap

Comment (JoanFM) [reply jina/helper.py:76]: No prob ;)

Comment (roccia) [reply jina/helper.py]: I got some errors on CI unit-test(py3.8)  `tests/unit/flow/test_flow.py:498:....... ERROR:root:Exception iterating responses: Address already in use` seems this is  the port issue,dose it always show up in CI? want to make sure it does not related with my commit

**Comment by roccia** [comment]
> Thank you very much for your contribution. Would it be possible for you to add also type hints for the return values?
> 
> Maybe you can use the help of mypy and MonkeyType?

no prob

File: jina/helper.py (hunk 1 above)
Comment (JoanFM) [reply jina/helper.py]: Hey do not worry, we are facing some stability issues related to this. It does not come from your PR.

**Comment by JoanFM** [comment]
> the type of variable `spec` in `_path_import` should be imported as `from importlib._bootstrap import ModuleSpec`,which violate `Access to a protected member _bootstrap of a module`, since it's never used in any place, so I think remove it would be a good choice.
> as for previous commit, I think it is better to leave the type hints for return value empty since some of the functions do not have a return value

I think you can use typehints with ' ' to avoid importing modules?

**Comment by roccia** [comment]
> I think you can use typehints with ' ' to avoid importing modules?

I  considered Any, and also thinked of ' ', but it seems just not right enough, since I already know it has a actual type

**Comment by JoanFM** [comment]
> > I think you can use typehints with ' ' to avoid importing modules?
> 
> I considered Any, and also thinked of ' ', but it seems just not right enough, since I already knew it has an actual type :(

I mean 'ActualType', is that what you mean it can't be done?

**Comment by roccia** [comment]
> ActualType

it can be done, the actual type is : 'ModuleSpec' , ModuleSpec is imported from  `importlib._bootstrap`,  while _bootstrap is a private method , s  I'm not sure if it is a best pratice to import Object from a private class

**Comment by JoanFM** [comment]
> > ActualType
> 
> it can be done, the actual type is : 'ModuleSpec' , ModuleSpec is imported from `importlib._bootstrap`, while _bootstrap is a private method , s I'm not sure if it is a best pratice to import Object from a private class

ok, leave it out maybe better.

**Comment by nan-wang** [comment]
recheckcla

**Comment by roccia** [comment]
> **Jina CLA check**
> 
> ❤️ Thank you for your pull request. It looks like this is your first contribution to an open source project maintained by Jina AI Limited. Before we can look at your pull request, we kindly ask that you sign our [Contributor License Agreement](https://github.com/jina-ai/jina/blob/master/CLA.md). You can sign it by commenting in format below.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_

**Comment by roccia** [comment]
**I have read the CLA Document and I hereby sign the CLA**

**Comment by nan-wang** [comment]
hi, @roccia The commit message is not properly formated, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
#553 :bookmark: Commit Message Guideline for the First Time Contributor
jina-ai/jina | Jun 21st | Added by GitHub
CONTRIBUTING.md
jina-ai/jina | Added by GitHub

**Comment by roccia** [comment]
> ge is

yes, I'm fixing it right now

**Comment by roccia** [comment]
> hi, @roccia The commit message is not properly formated, please refer to:
> 
> * [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
> * [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
>   #553 🔖 Commit Message Guideline for the First Time Contributor
>   jina-ai/jina | Jun 21st | Added by GitHub
>   CONTRIBUTING.md
>   jina-ai/jina | Added by GitHub

yes I'm fixing it right now

**Comment by roccia** [comment]
**I have read the CLA Document and I hereby sign the CLA**

**Comment by roccia** [comment]
_**I have read the CLA Document and I hereby sign the CLA**_

**Comment by hanxiao** [comment]
Thanks for your contribution ❤️ 

Oh btw, we have a slack channel for warm-heart contributors like you, where we discuss code, PR, and future of Jina. Strongly recommend to join us

- If you haven't joined our Slack, [click here to join](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w)
- 👨‍💻 👩‍💻 ➡️  https://jina-ai.slack.com/messages/contributors/

**Comment by roccia** [comment]
> Thanks for your contribution ❤️
> 
> Oh btw, we have a slack channel for warm-heart contributors like you, where we discuss code, PR, and future of Jina. Strongly recommend to join us
> 
> * If you haven't joined our Slack, [click here to join](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w)
> * 👨‍💻 👩‍💻 ➡️  https://jina-ai.slack.com/messages/contributors/

already joined

**Comment by alexcg1** [comment]
@all-contributors please add @roccia  for code

**Pull request merged**